Package.swift

```diff
@@ -30,9 +30,14 @@ let package = Package(
     ],
     targets: [
         .target(
-            name: "Proton",
+            name: "ProtonCore",
             dependencies: [],
-            path: "Proton/Sources"
+            path: "Proton/Sources/ObjC"         
```

Comment:
All ObjC code now needs to reside in this module. Also, it cannot use anything from Swift module.

Reply:
Out of curiosity, why didn't you put them in `ProtonCore` folder? Is it a limitation of SPM?

Reply:
it's not a SPM limitation. It could have been any folder. However, I wanted to clearly distinguish this as ObjC only.